libkbfs/root_metadata.go

```diff
@@ -1089,6 +1089,9 @@ func (rmds *RootMetadataSigned) MakeFinalCopy(
 	}
 	// Set the final flag.
 	newBareMd.SetFinalBit()
+	// Set the copied bit, so that clients don't take the ops and byte
+	// counts in it seriously.
+	newBareMd.SetWriterMetadataCopiedBit()
```

Comment:
how hard would it be to write a regression test? i.e. a test to make sure that a final MD copy is accepted by a client.

If this is hard, feel free to add a TODO/issue instead.